src/main/java/net/fosforito/partido/api/GroupsApi.java

```diff
@@ -38,29 +34,31 @@
   private final CurrentUserContext currentUserContext;
   private final HttpServletRequest request;
   private final EmailService emailService;
+  private final UTCDateService dateService;
 
   @Inject
   public GroupsApi(GroupRepository groupRepository,
                    UserRepository userRepository,
                    GroupService groupService,
                    CurrentUserContext currentUserContext,
                    HttpServletRequest request,
-                   EmailService emailService) {
+                   EmailService emailService, UTCDateService dateService) {
     this.groupRepository = groupRepository;
     this.userRepository = userRepository;
     this.groupService = groupService;
     this.currentUserContext = currentUserContext;
     this.request = request;
     this.emailService = emailService;
+    this.dateService = dateService;
   }
 
   /**
    * This method returns a list of all groups, a user has access to.
    * @return List with group objects
    */
   @GetMapping(
-      value = "/currentusergroups",
-      produces = MediaType.APPLICATION_JSON
+          value = "/currentusergroups",
```

Comment:
No prob. I'll try to stick to what is. ;)
Also, I'll look into this during the days to come, maybe today depending on work and a car I'm picking up. :)

Reply:
This indentation, is the same according to my IDE?

Reply:
Oh, you're right. The code change here on GitHub looked like the indentation changed but it seems that the indentation is generally not correct in the file. This will be changed with another issue regarding Checkstyle integration.